test/integration/nss-tests.sh

```diff
@@ -23,6 +23,7 @@ function pinentry() {
   cmd="$*"
   printf 'spawn %s\nexpect "Password or Pin"\nsend -- %s\\r\nexpect eof\n' \
 	  "$cmd" "$pin" | expect
+  exit $?
```

Comment:
This change seems bogus. Now the script as a whole exists after the first call to `pinentry`. The original code looks correct to me (`return $?` is implicit in bash).